rust/rsmgp-sys/src/edge/mod.rs

```diff
@@ -116,12 +116,12 @@ impl Edge {
         #[cfg(not(test))]
         assert!(
             !ptr.is_null(),
-            "Unable to make edge copy because edge pointer is null."
+            "Unable to create edge copy because the given pointer is null."
```

Comment:
If we would like to nitpick, then I would suggest `Unable to copy edge because the given pointer is null.`, but I am also fine with this message, if you like the current one more.

Reply:
I just realized this error message is in sync with the regarding `MgpError` so it is totally fine to leave this as is.